### PR TITLE
getting started improvements

### PR DESCRIPTION
**Pull request by nikolasburk**

### CODE REVIEW

File: content/300-accelerate/index.mdx [hunk 1]
```diff
@@ -4,7 +4,7 @@ metaTitle: 'Prisma Accelerate'
 metaDescription: 'Prisma Accelerate is a global database cache with built-in connection pooling that helps improve database performance in Serverless and Edge applications.'
 sidebar_position: 0
 hide_table_of_contents: false
-pagination_next: 'accelerate/what-is-accelerate'
+# pagination_next: 'accelerate'
```
Comment (jharrell) [inline content/300-accelerate/index.mdx]: meant to be commented?

File: static/_redirects [hunk 2]
```diff
@@ -511,7 +511,7 @@
 /concepts/overview/prisma-in-your-stack/graphql /docs/orm/overview/prisma-in-your-stack/graphql
 /concepts/overview/prisma-in-your-stack/rest /docs/orm/overview/prisma-in-your-stack/rest
 /accelerate/concepts /docs/accelerate
-/data-platform/accelerate/concepts /docs/accelerate/what-is-accelerate
+/data-platform/accelerate/concepts /docs/accelerate
```
Comment (jharrell) [inline static/_redirects:514]: I don't think we should remove existing destinations in our redirects. Open to be wrong though.

File: static/_redirects [hunk 3]
```diff
@@ -576,3 +576,7 @@
 /getting-started/setup-prisma/start-from-scratch/mongodb/next-steps-* /docs/getting-started/setup-prisma/start-from-scratch/mongodb/next-steps
 /getting-started/setup-prisma/add-to-existing-project/relational-databases/next-steps-* /docs/getting-started/setup-prisma/add-to-existing-project/relational-databases/next-steps
 /getting-started/setup-prisma/add-to-existing-project/mongodb/next-steps-* /docs/getting-started/setup-prisma/add-to-existing-project/mongodb/next-steps
+
+
+/accelerate/what-is-accelerate /docs/accelerate/
+/pulse/what-is-pulse /docs/pulse/
```
Comment (jharrell) [inline static/_redirects]: recommend moving these to above the `Dynamic redirects` section. This is due to a weirdness with CF handling redirects.

File: content/100-getting-started/01-quickstart.mdx [hunk 4]
```diff
@@ -427,6 +427,37 @@ If you want to move forward with Prisma ORM using your own PostgreSQL, MySQL, Mo
 
 Check out the [`prisma-examples`](https://github.com/prisma/prisma-examples/) repository on GitHub to see how Prisma ORM can be used with your favorite library. The repo contains examples with Express, NestJS, GraphQL as well as fullstack examples with Next.js and Vue.js, and a lot more.
 
+### Build real-time apps with Prisma Pulse
+
+[Prisma Pulse](/pulse) enables you to stream change events from your database in a type-safe way. Here are some example projects that show what you can build with it:
```
Comment (mhessdev) [inline content/100-getting-started/01-quickstart.mdx]: enables you to create applications that instantly react to changes in your database, allowing you to build type-safe real-time features and applications easily

File: content/100-getting-started/01-quickstart.mdx [hunk 5]
```diff
@@ -427,6 +427,37 @@ If you want to move forward with Prisma ORM using your own PostgreSQL, MySQL, Mo
...
+
+| Demo                                                                     | Description                                                              |
+| ------------------------------------------------------------------------ | ------------------------------------------------------------------------ |
+| [`starter`](https://github.com/prisma/prisma-examples/tree/latest/pulse/starter)                                             | A Prisma Pulse starter app                                               |
```
Comment (mhessdev) [inline content/100-getting-started/01-quickstart.mdx:436]: should we maybe add the techstack entierly? like nextjs, node, typescript etc?

File: static/_redirects (hunk 2 above)
Comment (mhessdev) [reply static/_redirects:514]: Not sure I understand. This is just changing the destination, not removing the redirect.

File: content/300-accelerate/index.mdx (hunk 1 above)
Comment (nikolasburk) [reply content/300-accelerate/index.mdx]: ah good catch, I commented it out to see if it was needed for some reason but don't think so, I guess it can be deleted entirely? 

```suggestion
```

File: content/400-pulse/index.mdx [hunk 6]
```diff
@@ -3,7 +3,7 @@ title: ''
 metaTitle: 'Prisma Pulse'
 metaDescription: 'Prisma Pulse enables real-time database events with type-safe Prisma Client subscriptions.'
 sidebar_position: 0
-pagination_next: 'pulse/what-is-pulse'
+# pagination_next: 'pulse'
```
Comment (nikolasburk) [inline content/400-pulse/index.mdx]:
```suggestion
```

File: static/_redirects (hunk 2 above)
Comment (nikolasburk) [reply static/_redirects:514]: Ah, good question and I'm not sure about best practices here!

My reasoning is that the `what-is-accelerate` and `what-is-pulse` pages are deleted. So redirecting to these pages doesn't make sense any more IMO. It would lead to a double-redirect and I'm not sure if that's desirable.

Do you have thoughts around this? Is a "double-redirect" ok? In that case, I'm happy to revert this, will trust your judgement here 🙂

File: static/_redirects (hunk 3 above)
Comment (nikolasburk) [reply static/_redirects]: ah thanks for the hint, moved ✅

File: content/100-getting-started/01-quickstart.mdx [hunk 7]
```diff
@@ -427,6 +427,33 @@ If you want to move forward with Prisma ORM using your own PostgreSQL, MySQL, Mo
 
 Check out the [`prisma-examples`](https://github.com/prisma/prisma-examples/) repository on GitHub to see how Prisma ORM can be used with your favorite library. The repo contains examples with Express, NestJS, GraphQL as well as fullstack examples with Next.js and Vue.js, and a lot more.
 
+### Build real-time apps with Prisma Pulse
+
+[Prisma Pulse](/pulse) enables you to stream change events from your database in a type-safe way. Here are some example projects that show what you can build with it:
+
+| Demo                                                                     | Description                                                              |
```
Comment (petradonka) [inline content/100-getting-started/01-quickstart.mdx:434]: Should we just link to the GitHub readme section to not have to maintain this list of examples here, or do you think it's better to be explicit, because the example descriptions give you an idea of Pulse also?

File: content/300-accelerate/800-examples.mdx [hunk 8]
```diff
@@ -0,0 +1,21 @@
+---
+title: 'Examples'
+metaTitle: 'Prisma Accelerate: Examples'
+metaDescription: 'Check out ready-to-run examples for Prisma Accelerate.'
+tocDepth: 3
+---
+
+## Prisma Accelerate examples
```
Comment (petradonka) [inline content/300-accelerate/800-examples.mdx:8]: I think we should just link to the examples repo instead of a new page, what do you think?

File: content/500-platform/10-about.mdx [hunk 9]
```diff
@@ -9,8 +9,8 @@ displayed_sidebar: platformSidebar
 
 The [Console](http://console.prisma.io/login) enables you to manage and configure your projects that use Prisma Data Platform (PDP) products, and helps you integrate them into your application:
 
-- [Accelerate](/accelerate/what-is-accelerate): Global database cache with scalable connection pooling
-- [Pulse](/pulse/what-is-pulse): A managed Change Data Capture (CDC) service that enables real-time database subscriptions
+- [Accelerate](/accelerate): Global database cache with scalable connection pooling
+- [Pulse](/pulse): A managed Change Data Capture (CDC) service that enables real-time database subscriptions
```
Comment (petradonka) [inline content/500-platform/10-about.mdx]: Can we rephrase this to not be the "managed CDC" positioning, but something more easily understandable?

File: content/100-getting-started/01-quickstart.mdx (hunk 7 above)
Comment (nikolasburk) [reply content/100-getting-started/01-quickstart.mdx:434]: > because the example descriptions give you an idea of Pulse also

Exactly, I think it's good that the examples are descriptive and give an idea of Pulse use cases.

I don't think that keeping the tables in sync is such a big issues (and also, there's not really a _risk_ when they get out of sync since nothing will "break").

File: content/300-accelerate/800-examples.mdx (hunk 8 above)
Comment (nikolasburk) [reply content/300-accelerate/800-examples.mdx:8]: You mean to place an external link in the sidenav? I _think_ this isn't possible... I had this [exact convo](https://prisma-company.slack.com/archives/C5Z9TH6N9/p1686304966190199) a while back already but back then the conclusion was that it's not possible; maybe this has changed with Docusaurus though?
 
cc @jharrell @carlagn

File: content/500-platform/10-about.mdx (hunk 9 above)
Comment (nikolasburk) [reply content/500-platform/10-about.mdx]: Updated to: 

- [Accelerate](/accelerate): Speeds up your queries with a global database cache with scalable connection pooling.
- [Pulse](/pulse): Enables you to build event-driven and real-time applications by letting you stream database change events in a type-safe way.

Does that work better?

File: content/300-accelerate/800-examples.mdx (hunk 8 above)
Comment (mhessdev) [reply content/300-accelerate/800-examples.mdx:8]: Yea i disagree with putting an external link in the side nav. Also I think having the examples be in the docs allows us more flexibility.

File: static/_redirects (hunk 2 above)
Comment (mhessdev) [reply static/_redirects:514]: Well if the pages are deleted than it should be like this 
`/data-platform/accelerate/concepts /docs/accelerate`
and
`/docs/accelerate/what-is-accelerate /docs/accelerate` 

That way the old deleted page gets redirected and the other redirect changes its location. Its only 1 redirect for each

Comment (nikolasburk) [reply static/_redirects:514]: > That way the old deleted page gets redirected and the other redirect changes its location. Its only 1 redirect for each

That was my thinking and I believe also what I did in the redirects file, right?

@jharrell do you have any concerns with this?

File: content/100-getting-started/01-quickstart.mdx (hunk 4 above)
Comment (nikolasburk) [reply content/100-getting-started/01-quickstart.mdx]:
```suggestion
[Prisma Pulse](/pulse) enables you to create applications that instantly react to changes in your database, allowing you to build type-safe real-time features and applications easily:
```

File: content/100-getting-started/01-quickstart.mdx (hunk 5 above)
Comment (jharrell) [reply content/100-getting-started/01-quickstart.mdx:436]: I think no, as it would make this table a bit harder to parse/understand.

File: static/_redirects (hunk 2 above)
Comment (jharrell) [reply static/_redirects:514]: Ah, I understand now. Thank you @nikolasburk, that makes a lot more sense. I think as you did it is correct.

File: static/_redirects [hunk 10]
```diff
@@ -528,10 +528,13 @@
 /platform/concepts/projects /docs/platform/about#project
 /platform/concepts/environments /docs/platform/about#environment
 /platform/concepts /docs/platform/about
-/pulse/concepts /docs/pulse/what-is-pulse
+/pulse/concepts /docs/pulse
 /more/roadmap /docs/orm/more/releases#roadmap
 /orm/overview/going-beyond /docs/orm/overview/beyond-prisma-orm
 
+/accelerate/what-is-accelerate /docs/accelerate/
```
Comment (jharrell) [inline static/_redirects]: IMO remove the space
```suggestion
/accelerate/what-is-accelerate /docs/accelerate/
```

File: content/300-accelerate/index.mdx (hunk 1 above)
Comment (jharrell) [reply content/300-accelerate/index.mdx]: Unfortunately, these are needed. See the "next" button on the bottom of the page: https://gse-improvements.docs-51g.pages.dev/accelerate

File: content/300-accelerate/800-examples.mdx (hunk 8 above)
Comment (petradonka) [reply content/300-accelerate/800-examples.mdx:8]: Hmh, okay. I indeed would've preferred just an external link. What flexibility do we gain here that's valuable?

**Review by petradonka — APPROVED** [review]
